packages/npm-dependencies-validation/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@sap-devx/npm-dependencies-validation",
+  "version": "1.3.0",
+  "private": true,
+  "description": "",
+  "author": "SAP SE",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SAP/app-studio-toolkit"
+  },
+  "main": "./dist/src/index",
+  "types": "./dist/src/index.d.ts",
+  "scripts": {
+    "test": "mocha",
```

Comment:
there should be a `ci` script which does the full build for this sub-package.
This is the script that will get called from the root of the monorepo
without it this sub-package will not get built